### ISSUE TITLE
[FR] - provide developer documentation

### ISSUE THREAD

**Issue by matiwinnetou** [adabox-aio/auth-server#2]
Introduction - why project is there
Hosting - how to host it, perhaps provide docker file for easy building and hosting
Developer Guide - present how to integrate with a demo website

**Comment by edridudi** [comment]
@matiwinnetou 
As you might know, we, developers are so bad when it comes to documentation :)
we prefer to spend our time building instead of documenting.

But,
you are most right. I will find the time to create a well-explained README with all the details.
Thank you.

**Comment by ccgarant** [comment]
Agree, and maybe a high-level concept of operations w/ pretty pictures to get an idea of how you think it should be used, so others can help contribute.  Great idea btw.
